bin/ci/before_install.sh

```diff
@@ -1,3 +1,5 @@
+gem uninstall -v '>= 2' -i $(rvm gemdir)@global -ax bundler || true
+gem install bundler -v '2.2.9'
```

Comment:
Code for this lifted from the Travis docs:

https://docs.travis-ci.com/user/languages/ruby/#bundler-20

(third bullet point after that link)